src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -641,28 +671,159 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
      * @brief Gets descriptive information for a cveid.
      *
      * @param cveId cveid to search.
+     * @param adpShortName short name of the ADP.
+     * @param adpSubShortName sub short name of the ADP.
      * @param resultContainer container struct to store the result.
      */
     void getVulnerabiltyDescriptiveInformation(const std::string_view cveId,
```

Comment:
This interface is to search for specific values ​​from the family columns of descriptions, apply business logic as to whether one thing or the other should not be part of this library.  Here we only have to model and obtain information.